ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/neon/(manage)/page.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import IntegrationsPage, {
+  type Publication,
+} from '@inngest/components/PostgresIntegrations/IntegrationPage';
+import { neonIntegrationPageContent } from '@inngest/components/PostgresIntegrations/Neon/neonContent';
+
+import { deleteConnection } from '../actions';
+
+export default function Page({ publication }: { publication: Publication }) {
+  return (
+    <IntegrationsPage
+      publication={publication}
+      content={neonIntegrationPageContent}
+      onDelete={deleteConnection}
```

Comment:
Prior iteration attempted to pass onDelete from a server component to a client, which next can't do. Instead wrap up the delete in an action so we can import it directly into a client component. Also moved the old server side integration fetch to a layout so we don't have to switch it to a client side fetch here. Also move that layout to it's own group so it doesn't also get picked up by the connect route and cause an infinite loop.  Fun stuff.